.github/workflows/smoke-checks.yml

```diff
@@ -113,7 +113,7 @@ jobs:
     - name: Run LLC Tests (Debug)
       env:
         DEVELOPER_DIR: "/Applications/Xcode_12.5.1.app/Contents/Developer"
-      run: bundle exec fastlane test device:"iPhone 12"
+      run: bundle exec fastlane test device:"iPhone 12" build_for_testing:true
```

Comment:
This will only run the LLC right? We need to run this for the UI SDK as well. 
Or we add another lane with `bundle exec fastlane test_ui device:"iPhone 12" build_for_testing:true`
Or we just build the demo app